packages/node_modules/@ciscospark/widget-recents/src/selector.js

```diff
@@ -118,55 +126,67 @@ const getRecentSpaces = createSelector(
 );
 
 
-const getRecentSpacesWithAvatarUrl = createSelector(
-  [getRecentSpaces, getAvatars],
-  (recentSpaces, avatars) => {
+const getRecentSpacesWithDetail = createSelector(
+  [getRecentSpaces, getAvatars, getCalls],
+  (recentSpaces, avatars, calls) => {
     const avatarItems = avatars.get('items');
-    if (avatarItems.count()) {
-      return recentSpaces.map((space) => {
-        const s = space;
+
+    return recentSpaces.map((space) => {
+      const s = space;
+      if (avatarItems.count()) {
         if (s.type === 'direct') {
           s.avatarUrl = avatarItems.get(s.toPersonId);
         }
         else {
           s.avatarUrl = avatarItems.get(s.id);
         }
-        return s;
-      });
-    }
-    return recentSpaces;
+      }
+      if (calls) {
+        s.call = calls.find((call) => {
+          const callSpaceId = call.instance.locus.conversationUrl.split('/').pop();
```

Comment:
Instead of deeply searching for this, which could cause issues later when locus changes, why don't we add `conversationUrl` or the `spaceId` to the top level `media`? Or we can add `locusUrl` to the top level space Record since all Spaces have locus associated with them.